### PR TITLE
Adding abbreviated status to agenda payload

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Dual Merge: [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/1980)

~- [ ] Postman tests for any new endpoints~ NA

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
made a mock change that im hoping is the test fix

File: talentmap_api/fsbid/services/agenda.py [hunk 1]
```diff
@@ -129,12 +141,14 @@ def fsbid_single_agenda_item_to_talentmap_single_agenda_item(data):
...
     legsToReturn.extend(sortedLegs)
+    statusFull = data.get("aisdesctext", None)
 
     return {
         "id": data.get("aiseqnum", None),
         "remarks": services.parse_agenda_remarks(data.get("aicombinedremarktext", '')),
         "panel_date": ensure_date(pydash.get(data, "Panel[0].pmddttm", None), utc_offset=-5),
-        "status": data.get("aisdesctext", None),
+        "status_full": statusFull,
+        "status_short": agendaStatusAbbrev.get(statusFull, None),
```
Comment (SidelineCory24) [inline talentmap_api/fsbid/services/agenda.py:151]: 🙌

File: talentmap_api/fsbid/services/common.py [hunk 2]
```diff
@@ -800,7 +800,7 @@ def get_aih_csv(data, filename):
         row.append(smart_str(pydash.get(record, "assignment.tod")))
         row.append(smart_str("=\"%s\"" % pydash.get(record, "assignment.grade")))
         row.append(panelDate)
-        row.append(smart_str(pydash.get(record, "status")))
+        row.append(smart_str(pydash.get(record, "status_full")))
```
Comment (SidelineCory24) [inline talentmap_api/fsbid/services/common.py:803]: nice catch on updating the export🐐

File: talentmap_api/fsbid/services/agenda.py [hunk 3]
```diff
@@ -118,7 +118,19 @@ def convert_agenda_item_query(query):
 
 
 def fsbid_single_agenda_item_to_talentmap_single_agenda_item(data):
-
+    agendaStatusAbbrev = {
+        "Approved": "APR",
+        "Deferred - Proposed Position": "XXX",
```
Comment (SidelineCory24) [inline talentmap_api/fsbid/services/agenda.py:123]: I like displaying the full status. Keeps consistency and we don't have to worry about changing `XXX` lol

**Comment by SidelineCory24** [comment]
Looks good to go! 🚀
Will approve/merge with FE PR

File: talentmap_api/fsbid/services/agenda.py (hunk 3 above)
Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/agenda.py:123]: 😂

**Review by SidelineCory24 — APPROVED** [review]
🚀

**Pull request merged**